CoreGraphNode/toc.md

```diff
@@ -0,0 +1,7 @@
+<div class="container">
```

Comment:
I think this file can be removed

Reply:
Removed, not even sure how it made its way in